docker/autoconfig.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
```

Comment:
Is this needed?

It seems to check bepasty configuration (which bepasty itself also does).

It has hardcoded defaults that may or may not be in sync with bepasty's internal defaults.

It adds new settings that (IIRC) bepasty does not have, like  BEPASTY_ADMIN_SECRET.